sdks/go/pkg/beam/core/runtime/exec/fn_test.go

```diff
@@ -150,7 +150,6 @@ func BenchmarkDirectCall(b *testing.B) {
 	for i := 0; i < b.N; i++ {
 		n = inc(n)
 	}
-	b.Log(n)
```

Comment:
We should keep the log statements. Otherwise, there is no observable output as 'n' is never used and a clever global optimizer could remove what we're trying to measure. It also allows manual inspection that something was computed when run. Is there a reason these were removed?

Reply:
That's a good point. I'll undo the removed lines for now, but I'll probably change them to not be output oriented going forward (in another CL).
They're spammy when running the benchmarks at present since the benchmark system ends up running the "test" several times until a certain time threshold is passed to settle on the value of N correctly.